src/transactions/ManageOfferOpFrameBase.cpp

```diff
@@ -517,7 +517,7 @@ ManageOfferOpFrameBase::doApply(AbstractLedgerTxn& ltxOuter)
 
         if (!creatingNewOffer || header.current().ledgerVersion >= 14)
         {
-            auto sourceAccount = loadAccount(ltx, getSourceID());
+            auto sourceAccount = loadSourceAccount(ltx, header);
```

Comment:
squash this into commit "Update ManageOfferOpFrameBase and OfferExchange for sponsorship" that introduced this bug